core/src/main/scala/com/pingcap/tispark/TiBatchWrite.scala

```diff
@@ -403,6 +404,13 @@ class TiBatchWrite(@transient val df: DataFrame,
     }
   }
 
+  private def checkHandleColumn(rdd: RDD[TiRow]): Unit = {
```

Comment:
maybe add `@throws(classOf[some error])`

Reply:
any test cases?

Reply:
the tests is in overflow test set